kong/plugins/http-log/sender.lua

```diff
@@ -58,7 +58,7 @@ local function send(self, bodies)
         return false
       end
     end
-
+    
```

Comment:
This patch introduced some undesired whitespaces here.

Reply:
Fixed.